src/message-backup/BackupClient.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Where message backups should be stored
+ */
+export enum BackupType {
+  none,
+  xmtp,
```

Comment:
nit: I thought enum members should always be capitalized but looks like we're already inconsistent with this repo.

Should we rename `xmtp` to `topicStore`? Kind of mirroring the:
```
export enum KeyStoreType {
  networkTopicStoreV1,
  static,
}
```

Reply:
My main thinking being an "xmtp" type backup doesn't tell me much as an external person. I'd probably think "yeah of course this is an XMTP backup", when really it should communicate the type/location of their "XMTP" backup. So like "xmtpTopicStore" or something

Reply:
I had the same confusion. In a world where there may be many different backupClients, specificity now will help.